tests/test_decoder.py

```diff
@@ -734,3 +734,111 @@ def test_parse_error():
 
     with pytest.raises(pvl.decoder.ParseError):
         pvl.load(io.BytesIO(b'foo'))
+
+
+@pytest.mark.parametrize(
```

Comment:
I've come up with 16 broken test labels. I have to spend more time figuring out edge cases and I will probably move these to their own test files later.